src/index.js

```diff
@@ -21,6 +21,12 @@ export class Animate extends React.Component {
     this.animating = false;
   }
 
+  componentWillMount() {
+    if (this.props.updateOnMount) {
```

Comment:
Yes, IIRC if you set `translateX` to `0` as a prop on the component, this should trigger an animation of `translateX` to `-1529.422` on mount. The reason the `0` needs to be set explicitly is because a spring animation needs a start value, and we can't always assume the start value is 0 (e.g. for scale it's 1).

Try that and let me know if it doesn't work.

I'm realizing now that setting `translateX` as a prop, and not as a `transform` inside the style object might feel awkward. I didn't realize this because I've been using https://github.com/steadicat/stylistic-elements, where the `translateX` prop makes sense. I'd be happy to look at a pull request that would check for start values inside a style object, or optionally receive a start value as props on `Animate`.